consensus/bor/bor.go

```diff
@@ -1191,11 +1194,15 @@ func (c *Bor) CommitStates(
 		}
 		stateSyncs = append(stateSyncs, &stateData)
 
-		if err := c.GenesisContractsClient.CommitState(eventRecord, state, header, chain); err != nil {
+		gasUsed, err := c.GenesisContractsClient.CommitState(eventRecord, state, header, chain)
```

Comment:
can we also log gas used per state sync?

Reply:
Sure !